benches/latency.rs

```diff
@@ -38,13 +39,11 @@ impl FutureService for Server {
 #[bench]
 fn latency(bencher: &mut Bencher) {
     let _ = env_logger::init();
+    let reactor = reactor::Core::new().unwrap();
     let addr = Server.listen("localhost:0".first_socket_addr(),
-                server::Options::default())
-        .wait()
+                server::Options::from(reactor.handle()))
```

Comment:
I think from is gone now. Can you fix this one so it builds? I've filed #105 to fix travis.